ansible/roles/etcd/tasks/main.yml

```diff
@@ -6,7 +6,11 @@
         state: latest
   notify:
     - restart etcd
-  when: not is_atomic
+  when: not is_atomic and ansible_distribution != "Ubuntu"
+
+- name: Install from github
+  include: github-release.yml
+  when: ansible_distribution == "Ubuntu"
```

Comment:
So I don't like that this is predicated on "Ubuntu".  Maybe a top level option on where to get etcd from?  And have it automatically selected if the user got it wrong?